03_letters/nicholeL/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(lettersFreq("aba")), "\n"))
+}
+
+func lettersFreq(s string) map[rune]int {
+	lettersMap := make(map[rune]int)
+	for _, letter := range s {
+		lettersMap[letter]++
```

Comment:
Great work realising this works even when the key doesn't exist yet (gotta love zero values) 👍